### DISCUSSION TITLE
Shopify plugin does not work

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#16311] [General]
Hey all,

Quick question, new to integrating Shopify into Craft CMS (v5.5.x). I've setup a custom app within Shopify and used the provided credentials to access the shop like so:

```
# Shopify
SHOPIFY_API_KEY="***********2e5c"
SHOPIFY_API_KEY_SECRET="***********db30"
SHOPIFY_ACCESS_TOKEN="***********8ac3"
SHOPIFY_HOST_NAME="**********.myshopify.com"
```

However, when I click webhooks in the navigation pane I get the following errror:

```
Shopify\Exception\RestResourceRequestException
REST request failed: "[API] Invalid API key or access token (unrecognized login or wrong password)"
```

I read on the Spotify Plugin page from Pixel & Tonic that I need to select a specific Webhook event version, however, this version is no longen available.

What am I missing here?

Cheers,
Allard

**Comment by brandonkelly** [comment]
Thanks for posting that! I’ve copied your post into a new issue on our Shopify plugin repo: https://github.com/craftcms/shopify/issues/128
